integration-tests/docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: "3"
 services:
 
   mimir:
-    image: grafana/mimir:latest
```

Comment:
got pranked here, the tag points now to an older version that does not support exponential histograms (it went back to 2.9.3)